azure-pipelines.yml

```diff
@@ -0,0 +1,40 @@
+# https://docs.microsoft.com/azure/devops/pipelines/languages/android
+jobs:
+  - job: gradle_junit_tests
+    steps:
+      - task: Gradle@2
+        inputs:
+          workingDirectory: espresso-server
+          gradleWrapperFile: espresso-server/gradlew
+          gradleOptions: '-Xmx3072m'
+          publishJUnitResults: true
+          testResultsFiles: '**/TEST-*.xml'
+          tasks: test
+  - job: sdk28_e2e_tests
+    pool:
+      vmImage: 'macOS-10.13'
```

Comment:
why not linux?

Reply:
https://docs.microsoft.com/en-us/azure/devops/pipelines/languages/android?view=azure-devops#test-on-the-android-emulator

`The Android Emulator is currently available only on the Hosted macOS agent`

Luckily Azure Pipelines provides concurrency of 10 agents, regardless of the platform.